sonarqube-build.gradle.kts

```diff
@@ -0,0 +1,39 @@
+import org.sonarqube.gradle.SonarQubePlugin
+
+buildscript {
+    repositories {
+        maven {
+            url = uri("https://maven.google.com")
+        }
+
+        val repository = if (project.hasProperty("qa")) "sonarsource-qa" else "sonarsource"
+        maven {
+            url = uri("https://repox.jfrog.io/repox/${repository}")
+
+            // The environment variables ARTIFACTORY_PRIVATE_USERNAME and ARTIFACTORY_PRIVATE_PASSWORD are used on QA env (Jenkins)
+            // On local box, please add artifactoryUsername and artifactoryPassword to ~/.gradle/gradle.properties
+            val artifactoryUsername = System.getenv("ARTIFACTORY_PRIVATE_USERNAME") ?: project.findProperty("artifactoryUsername") ?: ""
+            val artifactoryPassword = System.getenv("ARTIFACTORY_PRIVATE_PASSWORD") ?: project.findProperty("artifactoryPassword") ?: ""
+
+            if (artifactoryUsername is String && artifactoryUsername.isNotEmpty() && artifactoryPassword is String && artifactoryPassword.isNotEmpty()) {
+                credentials {
+                    username = artifactoryUsername
+                    password = artifactoryPassword
+                }
+            }
+        }
+    }
+    dependencies {
+        classpath("org.sonarsource.scanner.gradle:sonarqube-gradle-plugin:3.3")
```

Comment:
should this be the latest version "4.0.0.2929"

Reply:
I don't see that version here: https://repox.jfrog.io/ui/native/sonarsource/org/sonarsource/scanner/gradle/sonarqube-gradle-plugin/
Am I missing something?
Can we use "4.1.0.3103"?

Reply:
I'm not sure we can, as 4.0.0.2929 is the latest released and 4.1 is the one we're current;y working on.

Is it failing with 4.0.0.2929?